server/portal/apps/system_monitor/views.py

```diff
@@ -47,61 +44,30 @@ class System:
 
     def __init__(self, system_dict):
         try:
+            self.display_name = system_dict.get('display_name')
             self.hostname = system_dict.get('hostname')
-            self.display_name = system_dict.get('displayName')
-            if 'ssh' in system_dict.keys():
-                self.ssh = system_dict.get('ssh')
-            if 'heartbeat' in system_dict.keys():
-                self.heartbeat = system_dict.get('heartbeat')
-            if 'tests' in system_dict.keys():
-                self.status_tests = system_dict.get('tests')
-            if 'jobs' in system_dict.keys():
-                self.resource_type = 'compute'
-                self.jobs = system_dict.get('jobs')
-                self.load_percentage = system_dict.get('load')
-                if isinstance(self.load_percentage, (float, int)):
-                    self.load_percentage = int((self.load_percentage * 100))
-                else:
-                    self.load_percentage = None
-                self.cpu_count = system_dict.get('totalCpu')
-                self.cpu_used = system_dict.get('usedCpu')
+            self.resource_type = 'compute'
+            self.running = system_dict.get('running')
+            self.waiting = system_dict.get('waiting')
+            self.load = system_dict.get('load')
+            if isinstance(self.load, (float, int)):
+                self.load = int((self.load * 100))
             else:
-                self.resource_type = 'storage'
-                self.cpu_count = 0
+                self.load = 0
```

Comment:
The snippet system monitor uses the `load_percentage` field, so we'll need that term here too:
```
            if isinstance(self.load_percentage, (float, int)):
                self.load_percentage = int((self.load_percentage * 100))
            else:
                self.load_percentage = 0
```

Reply:
https://frontera-portal.tacc.utexas.edu/admin/djangocms_snippet/snippet/10/change/?_changelist_filters=q%3Dsystem